ddl/ddl_api.go

```diff
@@ -1745,3 +1749,24 @@ func validateCommentLength(vars *variable.SessionVars, comment string, maxLen in
 	}
 	return comment, nil
 }
+
+func checkModifyingDefinition(ctx sessionctx.Context, ident ast.Ident, originalColName model.CIStr) error {
```

Comment:
Since `tidb-server` is a concurrent program, it is highly possible while you running this function, other grouting may select data from the table you wanted to change. How can you establish the order of them? You may need add a variable in "sessioncontext" to let other grouting knowing you are changing table's schema.